app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -99,16 +99,14 @@ export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content }) =>
           </Row>
           <div className="list">
             {myReports.map((report) => (
-              <>
```

Comment:
@jdtoombs You need to test with your developer tools open more.  This is an easy error to catch.